code/game/objects/items/weapons/tape.dm

```diff
@@ -87,7 +87,7 @@
 				if(!can_place)
 					return
 
-				if(!H || !src || !H.organs_by_name[BP_HEAD] || !H.has_eyes() || H.glasses || (H.head && (H.head.body_parts_covered & FACE)))
+				if(!H || !src || !H.organs_by_name[BP_HEAD] || !H.has_eyes() || (H.head && (H.head.body_parts_covered & FACE)))
```

Comment:
But you still can't tape their mouth if they don't have eyes? >.>;

I get the feeling this if() was copied from blindfold taping.

Reply:
It was and I didn't notice that cause I was looking for 'glasses'